python/test/test_polars.py

```diff
@@ -347,6 +348,50 @@ def test_polars_context():
     )
 
 
+def test_polars_accuracy():
+    pl = pytest.importorskip("polars")
+    pl_testing = pytest.importorskip("polars.testing")
+
+    lf = pl.LazyFrame(schema={"A": pl.Int32, "B": pl.String})
+
+    context = dp.Context.compositor(
+        data=lf,
+        privacy_unit=dp.unit_of(contributions=1),
+        privacy_loss=dp.loss_of(epsilon=1.0),
+        split_evenly_over=2,
+        margins={
+            ("B",): dp.Margin(public_info="keys", max_partition_length=5),
+        },
+    )
+
+    expected = pl.DataFrame(
+        {
+            "column": ["len", "A"],
+            "aggregate": ["Len", "Sum"],
+            "distribution": ["Laplace", "Laplace"],
+            "scale": [8.0, 8.0],
+        }
+    )
+
+    query = (
+        context.query()
+        .group_by("B")
+        .agg(pl.len().dp.noise(), pl.col("A").fill_null(2).dp.sum((0, 3)))
+    )
+
+    actual = query.accuracy()  # type: ignore[union-attr]
+    pl_testing.assert_frame_equal(expected, actual)
+
+    expected = expected.with_columns(accuracy=23.965858188431927)
```

Comment:
Tests are failing here:
```
        expected = expected.with_columns(accuracy=23.965858188431927)
        actual = query.accuracy(alpha=0.05)  # type: ignore[union-attr]
>       pl_testing.assert_frame_equal(expected, actual)

test/test_polars.py:387: 
...
E       AssertionError: DataFrames are different (value mismatch for column 'accuracy')
E       [left]:  [23.965858188431927, 23.965858188431927]
E       [right]: [24.450243350374137, 24.450243350374137]
```
I don't know the big picture: Did this pass locally? Is it supposed to be deterministic?

Reply:
This started to fail after I adjusted the code to distinguish between continuous and discrete accuracy estimates- I think I mistakenly only committed this fix higher in the stack.